### DISCUSSION TITLE
(python3) add Python Scrips to path

### DISCUSSION THREAD

**Discussion by soredake** [chocolatey-community/chocolatey-packages#2039] [Ideas]
When installing packages without python installed with `/NoLockdown` you will get:
```
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: 'C:\\Python311\\Scripts\\jsonpointer'
Consider using the `--user` option or check the permissions.
```
When you will try to install them with `--user` option you will get:
```
  WARNING: The script normalizer.exe is installed in 'C:\Users\user\AppData\Roaming\Python\Python311\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script tqdm.exe is installed in 'C:\Users\user\AppData\Roaming\Python\Python311\Scripts' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  DEPRECATION: internetarchive is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
```

So maybe python3 package can add 'C:\Users\<user>\AppData\Roaming\Python\Python<version>\Scripts' to path automatically?

Winget package for example installs python in user mode and python installer will do this automatically, but in choco python is installed in system mode so python installer adds only `C:\Python<version>\Scripts\` to path

**Comment by AdmiringWorm** [comment]
Adding the user-specific scripts directory would only be possible for single-user systems. If you have multiple users installed, only the current user would have their user-specific directory in their PATH.

With that said, I would not mind a parameter being added to add it on the user path.

If you are willing to create a PR to add a parameter to the package to get this added, I can then make an issue that the PR can be targeted against (we don't accept PR targeting discussions).

Let me know if you want to create a PR for it (or if someone else is willing to do the work).
